### PR TITLE
crud for comments and restaurants

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: app/views/dishes/show.jbuilder [hunk 1]
```diff
@@ -1,3 +1,9 @@
 json.id @dish.id
 json.name @dish.name
-json.content @dish.content
\ No newline at end of file
+json.content @dish.content
+
+json.comments @posts do |comment|
```
Comment (h8rry) [inline app/views/dishes/show.jbuilder]: This should be for dishes, not posts

**Comment by ghost** [comment]
changed

File: app/views/dishes/show.jbuilder [hunk 2]
```diff
@@ -1,3 +1,9 @@
 json.id @dish.id
 json.name @dish.name
-json.content @dish.content
\ No newline at end of file
+json.content @dish.content
+
+json.comments @dishes do |comment|
```
Comment (h8rry) [inline app/views/dishes/show.jbuilder]: @dish.comments

**Comment by h8rry** [comment]
Looks alright :smile:

**Pull request merged**